src/App.jsx

```diff
@@ -91,21 +92,29 @@ const App = () => {
                   isSelected: prevColumns[idx + 1]?.selected === name,
                 }))
               : column.items,
-          })),
+          }))
         );
       })
       .finally(() => setLoading(false));
 
+    const dataPath = columnPaths.at(-1);
+    console.log("fetching data", dataPath);
+
     // Fetch data
-    fetchData(apiEndpoint, columnPaths.at(-1), blockHeight).then((response) => {
+    fetchData(apiEndpoint, dataPath, blockHeight).then((response) => {
       if (response) {
-        console.log("Data received from fetchData:", response.data);
-        setDataView(response.data.value);
+        console.log("Data received from fetchData:", response.data.value);
+        const take2 = storageHelper.unserializeTxt(response.data.value);
+        console.log("DEBUG take2", take2);
+        const obj = JSON.parse(response.data.value);
+        console.log("DEBUG obj", typeof obj, obj);
+        obj.values = obj.values.map((v) => JSON.parse(v));
+        console.log("DEBUG obj.values", obj.values);
+        setDataView(obj);
```

Comment:
Are these console.log's intentional? What's difficult to work out?

I pulled this down locally. For `fastUsdc` it doesn't seem to render keys that include JSON - only capData. Examples are the `fastUsdc` root  and `fastUsdc.status.${hash}`. I also noticed the old `p2p-agoric-vstorage-viewer` struggled with JSON as well.

Reply:
They were intentional while debugging. Looks like you were reviewing by commit. They're not in HEAD.

I don't see much value in cleaning those out of the commit history but would entertain the request

Reply:
I did it tho

Reply:
> Looks like you were reviewing by commit

Yep, thanks for removing, though.

> it doesn't seem to render keys that include JSON - only capData

Was this an observation you also had?

Reply:
yes. I want to solve this in client-utils `vstorage.js` or `vstorage-kit.js` by making them handle non-capdata JSON and even bare values